src/clients/c++/library/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ configure_file(request.h.in request.h @ONLY)
 configure_file(request_grpc.h.in request_grpc.h @ONLY)
 configure_file(request_http.h.in request_http.h @ONLY)
 
+if(${TRTIS_ENABLE_HTTP_V2})
```

Comment:
You should not have to add this again since you did it at the top level.

Reply:
You are right. Removed.